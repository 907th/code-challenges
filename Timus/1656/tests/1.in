3
170
170
180
175
170
175
175
170
175
