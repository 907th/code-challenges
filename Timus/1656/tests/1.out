170 175 170
175 180 175
170 175 170
